transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -440,22 +441,22 @@ object TransactorService {
   }
   
   def journalBlockReferenceToEvent(ref: Reference): JournalEvent = {
-    JournalEvent().withEvent(Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
+    JournalEvent(JournalEvent.Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
```

Comment:
Looks weird, is this the best way to use `oneOf`? Also slightly confusing because there's a `StateMachine.JournalEvent`